jest.config.js

```diff
@@ -16,8 +16,10 @@ module.exports = {
   collectCoverageFrom: [
     'src/**/*.{ts,tsx,js,jsx}',
     '!src/**/*.d.ts',
+    '!src/import_dir/*.ts',
   ],
   coveragePathIgnorePatterns: [
     'examples/',
+    'src/import_dir/',
```

Comment:
I wonder if just `import_dir` would work, if it matches anywhere in the string?

Reply:
Gonna give it a try.

Reply:
rip, ok just take out the `--coverage` I guess